test/testng/org/openjdk/jextract/test/api/TestAttributes.java

```diff
@@ -97,62 +95,4 @@ public void testB() {
         Declaration.Scoped d = parse("libAsmSymbol.h");
         validateHeader(d, false);
     }
-
-    private static  Constable getSingleValue(Declaration d, String name) {
-        List<Constable> values = d.getAttribute(name).get();
-        assertEquals(1, values.size());
-        return values.get(0);
-    }
-
-    @Test
-    public void testAddAttribute() {
```

Comment:
These tests were geared towards adding and removing attributes. Since the new attribute machinery is "add-only", these tests no longer made sense.